cmd/erasure-object.go

```diff
@@ -366,58 +346,15 @@ func (er erasureObjects) getObject(ctx context.Context, bucket, object string, s
 	return er.getObjectWithFileInfo(ctx, bucket, object, startOffset, length, writer, etag, opts, fi, metaArr, onlineDisks)
 }
 
-// getObjectInfoDir - This getObjectInfo is specific to object directory lookup.
-func (er erasureObjects) getObjectInfoDir(ctx context.Context, bucket, object string) (ObjectInfo, error) {
```

Comment:
Is this an old code, which means we are still able to HEAD or GET an empty file which has trailing slash in its name ?

Reply:
That will never happen @vadmeste the top level caller converts the / ending objects into __XL_DIR__